test.ps1

```diff
@@ -0,0 +1,7 @@
+Remove-Item -Recurse .\tools\NUnit.Runners.*
+.\tools\nuget install NUnit.Runners -Version 2.6.4 -OutputDirectory tools
```

Comment:
I recommend against running arbitrary binaries from the internet. It becomes an attack vector.

Nuget is likely to be installed (or easily installed) on a developer machine, why not rely on it being existing and in PATH? I certainly don't want a nuget copy for each project I work on.